### PR TITLE
Add tty password input

### PR DESCRIPTION
**Pull request by gguoss**

### CODE REVIEW

File: core/service/src/lib.rs [hunk 1]
```diff
@@ -112,6 +112,17 @@ pub struct TelemetryOnConnect<'a> {
 	pub executor: &'a TaskExecutor,
 }
 
+fn input_keystore_password() -> Result<String, String> {
+	let password =
+		rpassword::read_password_from_tty(Some("Password: ")).map_err(|e| format!("{:?}", e))?;
+	let confirmed_password = rpassword::read_password_from_tty(Some("Password again: "))
+		.map_err(|e| format!("{:?}", e))?;
+	if confirmed_password != password {
+		return Err("Password is not match".to_string());
```
Comment (arkpar) [inline core/service/src/lib.rs]:
```suggestion
		return Err("Password does not match".to_string());
```

File: core/service/src/lib.rs [hunk 2]
```diff
@@ -112,6 +112,17 @@ pub struct TelemetryOnConnect<'a> {
 	pub executor: &'a TaskExecutor,
 }
 
+fn input_keystore_password() -> Result<String, String> {
+	let password =
+		rpassword::read_password_from_tty(Some("Password: ")).map_err(|e| format!("{:?}", e))?;
+	let confirmed_password = rpassword::read_password_from_tty(Some("Password again: "))
```
Comment (arkpar) [inline core/service/src/lib.rs]:
```suggestion
	let confirmed_password = rpassword::read_password_from_tty(Some("Repeat password: "))
```

**Comment by gavofyork** [comment]
woulds be good to have the password confirmation prompt only when the key store doesn't yet exist. there's no need to have a confirmation except when creating the keystore in the first place.

**Comment by gavofyork** [comment]
@gguoss will need the Cargo.lock sorting out.

**Pull request merged**